apps/functions-subscription/src/adapters/azure/functions/subscriptions.ts

```diff
@@ -0,0 +1,76 @@
+import * as H from '@pagopa/handler-kit';
+import { pipe } from 'fp-ts/function';
+import * as RTE from 'fp-ts/ReaderTaskEither';
+import { httpAzureFunction } from '@pagopa/handler-kit-azure-func';
+import { NonEmptyString } from '@pagopa/ts-commons/lib/strings';
+import { Subscription } from '../../../generated/definitions/internal/Subscription';
+import { CreateSubscription } from '../../../generated/definitions/internal/CreateSubscription';
+import {
+  UserId,
+  TrialId,
+  Subscription as DomainSubscription,
```

Comment:
**nitpick**: How about using an alias for the API type instead of the domain type? This way, the imports would look like this:

```ts
import { Subscription as SubscriptionAPI } from '../../../generated/definitions/internal/Subscription';
import { ..., Subscription } from '../../../domain/subscription';
```

Reply:
✅